admin/src/views/Dashboard.vue

```diff
@@ -267,133 +269,112 @@
-<!--                      />-->
-<!--                    </v-col>-->
-
-<!--                    <v-col-->
-<!--                      cols="2"-->
-<!--                      class="text-right"-->
-<!--                    >-->
-<!--                      <v-icon class="mx-1">-->
-<!--                        mdi-pencil-->
-<!--                      </v-icon>-->
-<!--                      <v-icon-->
-<!--                        color="error"-->
-<!--                        class="mx-1"-->
-<!--                      >-->
-<!--                        mdi-close-->
-<!--                      </v-icon>-->
-<!--                    </v-col>-->
-<!--                  </v-row>-->
-<!--                </template>-->
-<!--              </v-card-text>-->
-<!--            </v-tab-item>-->
-<!--          </v-tabs-items>-->
-<!--        </base-material-card>-->
-<!--      </v-col>-->
+      <v-col
+        cols="12"
+        md="6"
+      >
+        <base-material-card
+          color="warning"
+          class="px-5 py-3"
+        >
+          <template v-slot:heading>
+            <div class="display-2 font-weight-light">
+              10 last modified items
+            </div>
+          </template>
+          <v-card-text>
+            <v-data-table
+              class="clickable"
+              hide-default-footer
+              :headers="itemsHeaders"
+              :items="data.items"
+              @click:row="editItem"
+            >
+              <template v-slot:item.creation_date="{ value }">
+                {{ value ? value.name : '' }}
+              </template>
+              <template v-slot:item.publish_state="{ value }">
+                {{ publishState(value) }}
+              </template>
+            </v-data-table>
+          </v-card-text>
+        </base-material-card>
+      </v-col>
+
+      <v-col
+        cols="12"
+        md="6"
+      >
+        <base-material-card
+          color="primary"
+          class="px-5 py-3"
+        >
+          <template v-slot:heading>
+            <div class="display-2 font-weight-light">
+              15 last activities
+            </div>
+          </template>
+          <v-card-text>
+            <v-data-table
+              hide-default-footer
+              :headers="activitiesHeaders"
+              :items="data.activities"
+            >
+              <template v-slot:item.created_at="{ value }">
+                <span class="text-no-wrap">
+                  {{ value }}
+                </span>
+              </template>
+              <template v-slot:item.user="{ value }">
+                {{ value.name }}
+              </template>
+              <template v-slot:item.auditable_type="{ item }">
+                <router-link
+                  class="text-no-wrap"
+                  v-if="item.id"
+                  :to="{ name: getEntityClass(item.auditable_type), params: { id: item.id } }"
+                >
+                  {{ getEntityClass(item.auditable_type) }} {{ item.id }}
```

Comment:
i think `item.id` here is the audit record it, maybe `auditable_id`?

Reply:
Yeah, you're right. I'm pushing the fix.